djangocms_versioning/static/djangocms_versioning/js/messages.js

```diff
@@ -0,0 +1,7 @@
+// Hide all messages after timeout to prevent overlap of breadcrumbs
+var djangoMessages = document.getElementsByClassName("messagelist");
+setTimeout(function(){
+    for (var i = 0; i < djangoMessages.length; i ++) {
+        djangoMessages[i].setAttribute('style', 'display:none !important');
+    }
+}, 3000);
```

Comment:
Would it be an improvement to put in a fade out, as opposed to setting display none?

Reply:
Good point. Replaced the display none with a fade out effect

Reply:
This will need to be initiated on document load to ensure that the dom has loaded the messagelist.

Reply:
This will need to be initiated on document load to ensure that the dom has loaded the messagelist.